### PR TITLE
[VEN-1419]: Psm/fork test

### PR DESCRIPTION
**Pull request by 0xlucian**
## Description

This PR adds for tests for PSM for USDC and USDT. It only covers straight cases to proof compatibility with the on-chain contracts. All edge cases and business logic of PSM is covered in the unit test suite

Resolves #[VEN-1419]

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: tests/hardhat/Fork/pegStabilityTest.ts [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+        });
+      });
+      describe("Swap", () => {
+        it("USDT -> VAI", async () => {
+          const token_amount = BigNumber.from(convertToUnit(1000, 18));
+          const fee = token_amount.mul(feeIn).div(BASIS_POINT_DIVISOR);
+          const vai_to_mint = token_amount.sub(fee);
+          await USDT.connect(usdtSigner).approve(psmUSDT.address, token_amount);
+          const tx = await psmUSDT.connect(usdtSigner).swapStableForVAI(USDT_HOLDER, convertToUnit(1000, 18));
+          await expect(tx).to.emit(psmUSDT, "StableForVAISwapped").withArgs(token_amount, vai_to_mint, fee);
```
Comment (chechu) [inline tests/hardhat/Fork/pegStabilityTest.ts:109]: Could we check the VAI balance of `usdtSigner` before and after the swap, and verify how much it was increased?

File: tests/hardhat/Fork/pegStabilityTest.ts [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+          const token_amount = BigNumber.from(convertToUnit(100, 18));
+          const token_amount_usd: BigNumber = token_amount.mul(price).div(mantissa_one);
+          console.log("USDT Price: " + price.toString());
+          await VAI.connect(vaiSigner).approve(psmUSDT.address, token_amount_usd);
+          const tx = await psmUSDT.connect(vaiSigner).swapVAIForStable(VAI_HOLDER, convertToUnit(100, 18));
+          await expect(tx).to.emit(psmUSDT, "VaiForStableSwapped").withArgs(token_amount_usd, 0, token_amount);
+        });
+      });
+    });
+    describe("Peg Stability USDC", () => {
```
Comment (chechu) [inline tests/hardhat/Fork/pegStabilityTest.ts:122]: Would it make sense to reuse the tests done for PSM-USDT? I think we could parametrise it enough to avoid so many "almost duplicated" code

**Review by chechu — APPROVED** [review]
